src/rad/resources/schemas/reference_files/matable-1.0.0.yaml

```
@@ -0,0 +1,255 @@
+%YAML 1.1
+---
+$schema: asdf://stsci.edu/datamodels/roman/schemas/rad_schema-1.0.0
+id: asdf://stsci.edu/datamodels/roman/schemas/reference_files/matable-1.0.0
+
+title: Multiple Accumulation Table Reference File Schema
+
+datamodel_name: MATableRefModel
+
+type: object
+properties:
+  meta:
+    allOf:
+      - $ref: asdf://stsci.edu/datamodels/roman/schemas/reference_files/ref_common-1.0.0
+      - type: object
+        properties:
+          reftype:
+            type: string
+            enum: [MATABLE]
+  guide_window_tables:
+    title: Guide Window Tables
+    description: |
+      Dictionary of guide window MA Tables keyed by their table ID
+      (a unique string identifier combining "GW" and a four-digit,
+      zero-padded integer between 1-9999)
+    type: object
+    patternProperties:
+      ^GW(\d{4})$:
```

```suggestion
      "^GW([0-9]{4})$":
```
To match the other patternProperties regexes let's quote these. The `\` causes yaml parsing issues when quoted but since `\d` is the same as `[0-9]` we can swap that in for the quoted version.

---

changed